openhands/sdk/conversation/conversation_stats.py

```diff
@@ -0,0 +1,162 @@
+import base64
+import pickle
+from threading import Lock
+
+from openhands.sdk.conversation.types import ConversationID
+from openhands.sdk.io.base import FileStore
+from openhands.sdk.llm.llm_registry import RegistryEvent
+from openhands.sdk.llm.utils.metrics import Metrics
+from openhands.sdk.logger import get_logger
+
+
+logger = get_logger(__name__)
+
+
+class ConversationStats:
+    def __init__(
+        self,
+        file_store: FileStore | None,
```

Comment:
Can we make this a Pydantic class AND part of `ConversationState`, so we can serialize it along with other things, instead of needing to manage these separately in another file?